tests/erlang_tests/test_min_heap_size.erl

```diff
@@ -28,12 +28,14 @@ start() ->
     receive
         ok -> ok
     end,
+    receive after 100 -> ok end,
```

Comment:
Did you try running `rebar3 fmt [...]`? I'm not sure the formatter would keep this.